src/demos/calcite-shell-rtl.html

```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE html>
```

Comment:
Can we get a simple demo of a shell by itself, or maybe with only a shell panel? 

I think the demo files should focus on showing off a single component, and only add as many supporting components as necessary (like a tip manager is pointless without a couple tips). This seems to be a full-blown example app. That has it's use, but it makes it hard to see what the shell by itself actually does. 

Perhaps this could go under an  "advanced demo" section. Might be better to split off a new section of demos that show this level of integration.